HeterogeneousCore/AlpakaTest/plugins/TestAlpakaAnalyzer.cc

```diff
@@ -0,0 +1,52 @@
+#include <cassert>
+#include <string>
+
+#include "DataFormats/PortableTestObjects/interface/TestHostCollection.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
```

Comment:
Is this `#include` needed (explicitly)? (it comes via `TestHostCollection` though)

Reply:
No (as per https://github.com/cms-sw/cmssw/pull/37716#discussion_r926170209). Removed.